examples/flutter_gallery/lib/demo/video_demo.dart

```diff
@@ -306,6 +307,13 @@ class _ConnectivityOverlayState extends State<ConnectivityOverlay> {
   @override
   void initState() {
     super.initState();
+    if (kIsWeb) {
```

Comment:
Add a TODO to remove this once the `connectivity` plugin is supported on web. We can support it with: https://api.dart.dev/stable/2.7.0/dart-html/Navigator/onLine.html

Reply:
Do we have a ticket for the connectivity plugin?

Reply:
Just added this: https://github.com/flutter/flutter/issues/46735

Reply:
Added a TODO, linked to the issue above.